llvm/test/CodeGen/RISCV/mul_sext_shl_constant.ll

```diff
@@ -0,0 +1,33 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 3
+; RUN: llc -mtriple=riscv64 < %s | FileCheck -check-prefixes=NO-ZBA %s
+; RUN: llc -mtriple=riscv64 -mattr=+zba < %s | FileCheck -check-prefixes=ZBA %s
+
+define ptr @g(ptr %0, i32 %1) {
```

Comment:
Can you add this test to the end of the existing test file which was modified by the change which added this code?  I don't think this needs it's own test file.